gandlf_generateMetrics

```diff
@@ -56,16 +56,18 @@ if __name__ == "__main__":
     parser.add_argument("-rawinput", "--rawinput", help=argparse.SUPPRESS)
 
     args = parser.parse_args()
-    assert args.config is not None, "Missing required parameter: config"
-    assert args.inputdata is not None, "Missing required parameter: inputdata"
+
+    # check for required parameters - this is needed here to keep the cli clean
+    for param_none_check in [args.inputDir, args.config]:
+        assert (
+            param_none_check is not None
+        ), f"Missing required parameter: {param_none_check}"
```

Comment:
@sarthakpati the issue is raised here:
```
  File "/GaNDLF/gandlf_generateMetrics", line 61, in <module>
    for param_none_check in [args.inputDir, args.config]:
AttributeError: 'Namespace' object has no attribute 'inputDir'
```
`generateMetrics` doesn't have `inputDir`, but `inputdata` instead

Reply:
Btw why do you check `None`-ness here? both config and inputdata are already marked as `required`:
```
    parser.add_argument(
        "-c",
        "--config",
        ...
        required=True,
        ...
```
So argparse should not allow empty values, isn't it?